packages/auth/src/authClient.ts

```diff
@@ -500,6 +500,7 @@ export class AuthClient {
       orgId,
       idToken,
       authType: type,
```

Comment:
as mentioned on `authSession.ts`, can we rename this to something like `sessionLengthMs`? assuming that the user will be passing in their desired session length and we are responsible for then calculating the expiration date from it